src/Cdn/README.md

```diff
@@ -45,7 +45,7 @@ module-version: 0.1.0
 # Normally, title is the service name
 title: Cdn
 subject-prefix: $(service-name)
-branch: 1278b951080f769d400bb5d03090de8bc21dc355
+branch: 20781a6d160ce5049d3a481ccac07b891700b7a3
```

Comment:
this won't work when you specified a certain branch at line 38 and line 39

Reply:
Fixed, thanks.